modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -99,7 +99,7 @@ spec:
 <3> Replace `<base64-of-pwd>` with the `base64` string of the password.
 <4> Replace `<NIC1-mac-address>` with the MAC address of the bare metal node's first NIC. See the BMC addressing section for additional BMC configuration options. Replace `<protocol>` with the BMC protocol, such as IPMI, RedFish, or others.
 <5> Replace `<bmc-ip>` with the IP address of the bare metal node's baseboard management controller.
-<6> Optional. You can set the `networkConfig` configuration option to configure host network interfaces. See "(Optional) Configuring host network interfaces in the `install-config.yaml` file" in the "Setting up the environment for an OpenShift installation" section for configuration details.
+<6> Optional. You can set the `networkConfig` configuration option to configure host network interfaces. See link:https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-host-network-interfaces-in-the-install-config.yaml-file_ipi-install-installation-workflow[(Optional) Configuring host network interfaces in the install-config.yaml file]  
```

Comment:
I would remove configuration since you say "to configure host..."
```suggestion
<6> Optional. You can set the `networkConfig` option to configure host network interfaces. See link:https://docs.openshift.com/container-platform/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-host-network-interfaces-in-the-install-config.yaml-file_ipi-install-installation-workflow[(Optional) Configuring host network interfaces in the install-config.yaml file]  
```